conanfile.py

```diff
@@ -101,6 +101,10 @@ def build_requirements(self):
         if self.options.build_docs:
             self.build_requires("doxygen/1.9.4") # doxygen/1.9.5 will update dependency on zlib/1.2.12 to zlib/1.2.13
 
+        if self.settings.compiler in ["gcc", "clang"]:
```

Comment:
```suggestion
        if self.settings.os == "Linux" and self.settings.compiler in ["gcc", "clang"]:
```

Although not tested yet, this could be for GCC under Cygwin or MinGW or Clang under Windows so it make sense to restrict this to Linux as these are the only environments we've tested.

Further down the line, we may move this into a function `checkMoldIsSupport` as the logic gets more complex.

Reply:
Clear, I agree. Thanks.